packages/system/src/Location/ListView/api.ts

```diff
@@ -78,6 +79,24 @@ export const useLocationUpdate = (): UseMutationResult<
   );
 };
 
+export const useLocationDelete = (): UseMutationResult<
+  DeleteLocationMutation,
+  unknown,
+  Location,
+  unknown
+> => {
+  const queryClient = useQueryClient();
+  const { api } = useOmSupplyApi();
+  const { storeId } = useQueryParams({ initialSortBy: { key: 'id' } });
+  return useMutation(
+    async (location: Location) =>
+      api.deleteLocation({ input: { id: location.id }, storeId: storeId }),
+    {
+      onSettled: () => queryClient.invalidateQueries(['location', 'list']),
```

Comment:
I wonder if this should be just `['location']` which would also invalidate some query like `['location', id]`

Reply:
cool - might as well